package.json

```diff
@@ -17,7 +17,7 @@
     "@babel/cli": "^7.5.5",
     "@babel/core": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
-    "chromedriver": "^78.0.0",
+    "chromedriver": "^79.0.0",
```

Comment:
The chromedriver recently got updated to version 80, but this has already been reflected in the master, so if you rebase, you should get the changes automatically.